src/query/query_manager.c

```diff
@@ -1397,16 +1397,26 @@ xqmgr_execute_query (THREAD_ENTRY * thread_p, const XASL_ID * xasl_id_p, QUERY_I
    * the XASL cache entry of the target query that is obtained from the XASL_ID, because all results of the query with
    * different parameters (host variables - DB_VALUES) are linked at the XASL cache entry.
    */
-  params.size = dbval_count;
-  params.vals = dbvals_p;
-
   if (copy_bind_value_to_tdes (thread_p, dbval_count, dbvals_p) != NO_ERROR)
     {
       goto exit_on_error;
     }
 
   if (qmgr_is_allowed_result_cache (*flag_p))
     {
+      params.size = dbval_count;
+      params.vals = NULL;
+
+      if (dbval_count)
```

Comment:
혹시 -1과 같은 값이 설정되어 있을 수도 있으니 명확하게 하는 것이 좋겠습니다.
```suggestion
      if (dbval_count > 0)
```

Reply:
dbval_count > 0으로 수정하겠습니다.